py_gtfs_rt_ingestion/lib/convert_gtfs.py

```diff
@@ -53,22 +53,22 @@ class GtfsConverter(Converter):
     Converter for GTFS Schedule Data
     """
 
-    def convert(self, files: list[str]) -> list[tuple[str, pyarrow.Table]]:
-        process_logger = ProcessLogger("convert_gtfs", filecount=len(files))
-        process_logger.log_start()
+    def __init__(self, config_type: ConfigType) -> None:
```

Comment:
new API for Converter requires a slight change here.

Reply:
[link](https://github.com/mbta/lamp/pull/69/commits/c427093a41ccaa8dba33659a5dd37e7b3272fff9#diff-ed3e7f70fabd180d3da9a1de30941921944bfe3957f9b9f25c825158ee031055R82-R118) to api change.